packages/material-ui/src/Slider/Slider.js

```diff
@@ -181,8 +181,8 @@ export const styles = theme => ({
     width: '100%',
     height: 2,
     borderRadius: 1,
-    backgroundColor: 'currentColor',
-    opacity: 0.38,
+    backgroundColor:
+      theme.palette.type === 'light' ? theme.palette.grey[400] : theme.palette.grey[600],
```

Comment:
The previous values were correct.

Reply:
Hi
Yes, I put these in to show how it could look.
The previous use of opacity creates some issues with  inverted and will not work. I played with a few options but it created unwanted complexity. Any suggestions on this ?

Reply:
As far as I remember, I went with the opacity to prioritize overrides simplicity.

Reply:
It also follows a common pattern in the spec, and the implementation in MCW, in that the track reflects the background shade. I can see how this would be difficult with the inverted track. 🤔

Reply:
Range/Inverted is the remaining problem, I can work-around the rest just by resizing track/rail.
To fix for Range/Inverted we could refactor and have separate start/end/track elements so the track does not have a rail beneath it and then correctly gets the color based on the background underneath ?